tsconfig.build.json

```diff
@@ -0,0 +1,23 @@
+{
+  "exclude": [
+    "**/*.test.ts",
+    "**/*.stub.ts",
+    "node_modules",
+    "dist"
+  ],
+  "extends": "@tsconfig/node14/tsconfig.json",
+  "compilerOptions": {
+    "moduleResolution": "node",
+    "noImplicitAny": true,
```

Comment:
not needed, see node14/tsconfig.json & strict=true https://www.typescriptlang.org/tsconfig#noImplicitAny

Reply:
```strict``` is now enabled